index/templates/index/navigation_autocomplete.html

```diff
@@ -0,0 +1,32 @@
+{% if problems %}
```

Comment:
Fine with this.
If having performance issue, I think we can just add license in the README.md.
Just decreasing some security